### ISSUE TITLE
lora with hunyuan inference issue

### ISSUE THREAD

**Issue by syyxsxx** [a-r-r-o-w/finetrainers#159]
### System Info / 系統信息

diffusers: from source

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

i trained the lora with hunyuan model by my custom dataset
But the output video is black screen, like this:
https://storage.googleapis.com/trimed_data/out/out1.mp4
this is my inference code:
```
import torch
from diffusers import HunyuanVideoPipeline

import torch
from diffusers import HunyuanVideoPipeline, HunyuanVideoTransformer3DModel
from diffusers.utils import export_to_video

model_id = "hunyuanvideo-community/HunyuanVideo"
transformer = HunyuanVideoTransformer3DModel.from_pretrained(
            model_id, subfolder="transformer", torch_dtype=torch.bfloat16
            )
pipe = HunyuanVideoPipeline.from_pretrained(model_id, transformer=transformer, torch_dtype=torch.float16)
pipe.load_lora_weights("./pytorch_lora_weights.safetensors", adapter_name="hunyuanvideo-lora")
pipe.set_adapters(["hunyuanvideo-lora"], [0.6])
pipe.vae.enable_tiling()
pipe.to("cuda")
prompts = [
"JD_STYLE, From ‘The Nightmare Before Christmas’, close up shot of Jack Skellington, skeletal figure, wearing a tattered red santa claus costume, singing while sitting on the book of winged angel stone statue. Slowly zoom out shot of Jack Skellington and the winged angel stone statue surrounded by snowy tombstones and explosive fire burning torches illuminated the dark graveyard. Heavy fog is blowing from the right.",
"JD_STYLE, From ‘The Nightmare Before Christmas’, close up slow arc right shot of Jack Skellington, skeletal figure, wearing a tattered red santa claus costume, singing while sitting on the book of winged angel stone statue.  Slowly zoom out shot of Jack Skellington and the winged angel stone statue surrounded by snowy tombstones and fire burning torches illuminated the dark graveyard",
"JD_STYLE, From ‘The Nightmare Before Christmas’, Close up shot of Jack Skellington wearing signature pinstripe suit and blue party hat standing in a snow covered graveyard with crosses and a female stone statue in background. Snow is falling down. Jake is reaching out to touch the falling snowflakes with both hands. Sally is standing next to Jack, looking at Jack and smiling."
]

i = 0
for p in prompts:
    output = pipe(
    prompt=p,
    height=432,
    width=768,
    num_frames=49,
    num_inference_steps=30,
    ).frames[0]
    out = 'out' + str(i) + '.mp4'
    export_to_video(output, out, fps=15)
    i = i + 1
```
How can I debug this issue?

### Expected behavior / 期待表现

issue be solved

**Comment by generalsvr** [comment]
Try to update pytorch to 2.5.1, the issue with black screen should be solved

**Comment by a-r-r-o-w** [comment]
Training and inference should both be done in torch 2.5.1 for it to work. Any version lower than that will lead to black video or nan loss. Sorry for the inconvenience - we should have highlighted this information better

**Comment by a-r-r-o-w** [comment]
Marking as closed because that should probably fix it. If not, please feel free to re-open
